src/services/api/handlers/getAttachmentUrl.ts

```diff
@@ -43,8 +48,13 @@ export const handler = async (event: APIGatewayEvent) => {
       });
     }
 
+    const allAttachments = [
+      ...results.hits[0]._source.attachments,
+      ...results.hits[0]._source.raiResponses.map((R) => R.attachments).flat(),
```

Comment:
Are we making the capital letters a convention for iterators. If so I'm totally cool with that

Reply:
lol i guess so. its not a horrible pattern

Reply:
my ode to @pkim-gswell